src/test/collectionHandlingFunctions.php

```diff
@@ -0,0 +1,304 @@
+<?php
+require "../collectionHandlingFunctions.php";
+use PHPUnit\Framework\TestCase;
+
+class CollectionHandlingFunctions extends TestCase 
```

Comment:
maybe collection should be lowercase c for function name

Reply:
done!